### PR TITLE
footer -link-added and solved conflicts

### PR DESCRIPTION
**Pull request by Varsani2520**
### Description
i solved the footer-link conflicts due to the production 
[Provide a brief overview of the changes introduced by this pull request.]

### Fixes

i solved the PR #55 

### Type of PR

- [x] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Refactoring
- [ ] Other (specify): _______________

### Checklist
- [x] I have read and followed the [Contribution Guidelines](https://github.com/omrajsharma/bigohhh.com/blob/main/Contribution.md).
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, snapshots, and videos after making the changes.
- [x] I have not borrowed code without disclosing it, if applicable.
- [x] This pull request is not a Work In Progress (WIP), and only completed and tested changes are included.
- [x] I have tested these changes locally.

### CODE REVIEW

**Comment by Varsani2520** [comment]
Hey @omrajsharma  I solved the conflict ,now you can merged it

File: client/src/components/footer/footer-navigation.tsx [hunk 1]
```diff
@@ -8,24 +8,24 @@ import { FooterSectionTitle } from '@/components/footer'
...
 const pageMenu: Array<Navigation> = [
-  { label: 'Home', path: '#' },
-  { label: 'Courses', path: '#' },
-  { label: 'Testimonial', path: '#' },
-  { label: 'Mentor', path: '#' },
+  { label: 'Home', path: '/' },
+  { label: 'Courses', path: '/popular-courses' },
+  { label: 'Testinomial', path: '/testinomial' },
+  { label: 'Mentor', path: '/mentor' },
 ]
```
[the remark this answers is not in the document]
Comment (Varsani2520) [reply client/src/components/footer/footer-navigation.tsx:22]: @omrajsharma ,i am  able to create a router based page ,if you want to then you can call me ,i make it 
and your website made  fully functionality 
look like a proffesional

**Comment by omrajsharma** [comment]
Can you introduce these routes in to the project ?

**Comment by omrajsharma** [comment]
@Varsani2520 please test your changes

**Comment by omrajsharma** [comment]
currently new feature on these routes are released, please check that as well.

**Comment by omrajsharma** [comment]
@Varsani2520 any update?

**Review by Varsani2520 — COMMENTED** [review]
add routes

**Pull request closed without merging**